js/main.js

```diff
@@ -0,0 +1,23 @@
+const buttons = document.querySelector('.hamburger');
+const menu = document.querySelector('.mobile-menu');
+const menuIcon = document.querySelector('.hamburger__close-icon');
+const hamburger = document.querySelector('.hamburger-button');
+const mIcons = document.querySelectorAll('mobile-menu');
+
+function toggleMenu() {
+  if (menu.classList.contains('showMenu')) {
+    menu.classList.remove('showMenu');
+    menuIcon.style.display = 'none';
+    hamburger.style.display = 'block';
+  } else {
+    menu.classList.add('showMenu');
+    menuIcon.style.display = 'block';
+    hamburger.style.display = 'none';
+  }
+}
```

Comment:
[OPTIONAL] Well done here 😉 but this could be improved, here's an improved version of the code that uses a helper function to toggle the menu class and update the display of the menu icon and 🍔  hamburger:
```
function toggleMenu() {
  toggleClass(menu, 'showMenu');
  toggleDisplay(menuIcon);
  toggleDisplay(hamburger);
}

function toggleClass(element, className) {
  if (element.classList.contains(className)) {
    element.classList.remove(className);
  } else {
    element.classList.add(className);
  }
}

function toggleDisplay(element) {
  if (element.style.display === 'none') {
    element.style.display = 'block';
  } else {
    element.style.display = 'none';
  }
}
```
The `toggleClass` function takes an element and a class name as arguments, and toggles the presence of the class on the element.

The `toggleDisplay` function takes an element as an argument, and toggles its display property between 'none' and 'block'.

The `toggleMenu` function then calls these two helper functions to toggle the menu class and update the display of the menu icon and hamburger.

Reply:
yeah thats good